manifests/config.pp

```diff
@@ -1,7 +1,7 @@
 # Class: bind::config
 #
 # This class manages bind configuration
-class bind::config {
```

Comment:
It is not the good way to pass this parameter. You should add it as attribute and set when needed

Reply:
I do not understand what you mean,
i passed this parameter with default value as it was before 'bind/named.conf.options.erb'

Reply:
sorry, just You need to put a default value in order to avoid side effect. It may be a blank one:
$bind_options_template = ''